### DISCUSSION TITLE
Link in table

### DISCUSSION THREAD

**Discussion by techno-disaster** [onweru/compose#44] [Q&A]
Is there any way I can have a clickable link like [this](https://example.com), inside a table value?

**Comment by onweru** [comment]
you mean like links in this table https://www.markdownguide.org/basic-syntax/#characters-you-can-escape?

**Reply by techno-disaster** [reply]
Yes, I tried adding something like that in projects.csv but it just reads the content and does not render a link. It works fine if you use a more traditional pure markdown way like 
``` 
| animal | sound |
|--------|-------|
| dog    | meow  |
| cat    | woof  |

```

**Reply by onweru** [reply]
Currently the content from your csv is not processed as markdown. It’s just plain text

**Reply by josehelps** [reply]
@onweru do you think there is any way to inject a link into the rendered table? I understand it is rendered in plaintext but would be nice to have it processes markdown (for adding a URL or formatting).

**Reply by josehelps** [reply]
Looking into this a bit deeper, is here where that change can be done? https://github.com/onweru/compose/blob/master/layouts/shortcodes/chart.html#L27-L51

**Reply by josehelps** [reply]
I made a fork of the template and was able to apply markdown on the csv:
https://github.com/onweru/compose/compare/master...josehelps:compose:master

![image](https://user-images.githubusercontent.com/1476868/218812899-7f66218d-0ce5-48e7-9e10-f73149c96318.png)

as you can see `project6` is a URL that in the project.csv is listed as:

![image](https://user-images.githubusercontent.com/1476868/218813173-f6b60933-9be0-47f6-a3db-1b62cf33f5d7.png)

**Reply by ksclarke** [reply]
Was this feature added back into the upstream project? I'm trying to do this, but it doesn't seem to work.

**Comment by techno-disaster** [comment]
Also is there any way to disable the next and previous buttons? @onweru

**Reply by onweru** [reply]
You can clone the theme and edit those parts out. I will add knobs for that later when I get some time
